lib/gcloud/storage/connection.rb

```diff
@@ -285,7 +285,7 @@ def download_file bucket_name, file_path
 
       ##
       # Updates a file's metadata.
-      def patch_file bucket_name, file_path, options = {}
+      def patch_file bucket_name, file_path, options = {}, body_options = {}
```

Comment:
I would prefer if `options` and `body_options` were the same hash. Having two makes it more difficult to use.

```ruby
def patch_file bucket_name, file_path, options = {}
```